### PR TITLE
Introduce staticRate order creation strategy

### PR DESCRIPTION
**Pull request by mdyring**
Seems useful to be able to define a static interest rate as well (perhaps used in conjunction with lowerRateWithTime to slowly move to top)

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -150,6 +150,9 @@ function manage(exchangeName, handle, currency, settings, cb) {
     function(cb) {
 	// 5. find desired rate
 	var strategy = strategies.createLoan.topOfTheBook;
+	if (settings.rateCreationStrategy.name === 'staticRate') {
+	    strategy = strategies.createLoan.staticRate;
+	}
 	if (settings.rateCreationStrategy.name === 'percentDepth') {
```
Comment (anders94) [inline app.js]: can we make this an `else if`?

**Review by anders94 — COMMENTED** [review]
Thanks - this makes good sense. Happy to merge once we make the slight `if` -> `else if` cosmetic change.

File: app.js (hunk 1 above)
Comment (mdyring) [reply app.js]: Sure - done

**Comment by anders94** [comment]
Thanks for the help!

**Pull request merged**